docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -121,4 +122,34 @@ Output directory defaults to `/tmp/<random_uuid>` but you can point it to a Azur
             value_type: swap
             value: "PowerState/running"
 
-More details on Blob Storage output are at :ref:`azure_bloboutput`
\ No newline at end of file
+More details on Blob Storage output are at :ref:`azure_bloboutput`
+
+
+Event Grid Functions
+####################
+
+Currently, support for event grid functions is at the subscription level and can listen to write and delete
+events. When deploying an event grid function, an Event Grid Subscription is created that triggers the Azure Function
+when any event is triggered in the subscription. Cloud custodian filters to the events you passed to your policy and
```

Comment:
it sounds like this is subscription mgmt ops of the set from here ? https://docs.microsoft.com/en-us/azure/event-grid/overview

was also considering arm events, but subscription is a good base. at some point ofc we'd want to support any filtering mechanism thats supported on the event source to avoid the extraneous func execs.

Reply:
These are ARM events emitted across an entire subscription.
https://docs.microsoft.com/en-us/azure/event-grid/event-schema-subscriptions